programs/ziti-prox-c/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 project(ziti-prox-c)
 
-add_executable(ziti-prox-c proxy.c)
+add_executable(ziti-prox-c proxy.c ${ZITI_HEADER_FILES})
```

Comment:
not needed?

Reply:
it's not needed for things to compile properly but without this entry the header files are buried in visual studio. This pic shows before the change (left) and after (right)

![image](https://user-images.githubusercontent.com/46322585/72559350-98d8e580-3872-11ea-8a91-e2a0a515fd82.png)

Reply:
still seems incorrect, an application that uses SDK won't add that or any other dependency headers (libuv, etc)